common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3219,12 +3219,63 @@
     },
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_TABLE_OPERATION" : {
```

Comment:
Does this duplicate `UNSUPPORTED_FEATURE.TABLE_OPERATION`?

Reply:
This is a difficult problem. 
If we merge `UNSUPPORTED_FEATURE.TABLE_OPERATION` to `UNSUPPORTED_TABLE_OPERATION.WITH_SUGGESTION`, 
we need to assign a new suitable subclass name for it, because it is different from `UNSUPPORTED_TABLE_OPERATION.WITH_SUGGESTION`'s suggestion information.

Reply:
I see. Let's leave this as you propose in the PR.

Reply:
This is too general, but what we actually report is relation type mismatch. How about
```
"UNEXPECTED_RELATION_TYPE": "<name> is a <actualType>. '<cmd>' expects a <expectedType>."
"subClass": {
  "WITHOUT_SUGGESTION": "",
  "WITH_ALTER_COMMAND_SUGGESTION": "Please use ALTER <expectedType> instead."
}
```